lib/config/accountsConfig.js

```diff
@@ -0,0 +1,78 @@
+var mySubmitFunc = function(error, state){
+  if (!error) {
```

Comment:
Do we want to handle errors in any specific way? If so, this should be:

`if (error)`

and the error case should be handled and a return should follow it.